ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/model/enums/Gender.java

```diff
@@ -9,14 +9,10 @@
 @AllArgsConstructor
 public enum Gender implements HasLabel {
 
-    @JsonProperty("male")
-    MALE("Male"),
-
-    @JsonProperty("female")
-    FEMALE("Female"),
-
-    @JsonProperty("notGiven")
-    NOT_GIVEN("Not given");
+    MALE("male"),
+    FEMALE("female"),
+    NOT_GIVEN("notGiven");
```

Comment:
If we use the label as dual purpose, Json value and CCD user interface label, it does remove the flexibility.

Reply:
Is see, so this option would now show in the UI as `notGiven` rather than `Not given`? I'll undo it if so

Reply:
Yes that is correct